### ISSUE TITLE
[Question] [LineChart] Its possible to know the barIndex in getTouchedSpotIndicator 

### ISSUE THREAD

**Issue by LuisCampos97** [imaNNeoFighT/fl_chart#466]
Hi again @imaNNeoFighT,

I want to customize the point when I touch, for each of the lines.
For line 0 I want the blue point, for line 1 I want the green point. 

**Is it possible to access the line index ??**

![image](https://user-images.githubusercontent.com/33434500/95491291-51aafb80-0991-11eb-8a5a-d014b05b4445.png)

I tried it this way but it gives me the error I mentioned in the code.

![image](https://user-images.githubusercontent.com/33434500/95491681-da299c00-0991-11eb-8e86-c7c350093818.png)

### Congratulations on the library once again.

**Versions**
**Flutter version:** 1.22.0
**FlChart version:** 0.12.0

**Comment by jesusmartinoza** [comment]
Hi @LuisCampos97 you are on the right way.

To customize the FlLine and FlDotData according to index you can do something like this.

```dart
getTouchedSpotIndicator: (LineChartBarData barData, List<int> spotIndexes) {
          var index = spotIndexes.first; // Get index of touched point

          return spotIndexes.map((e) {
            return TouchedSpotIndicatorData(
              FlLine(color:  index % 2 == 0 ? Colors.red : Colors.blue),
              FlDotData(
                show: show,
                getDotPainter: (spot, percent, barData, index) => FlDotCirclePainter(
                  radius: 3,
                  color: index % 2 == 0 ? Colors.red : Colors.blue), // Change color according to your own logic 
                  strokeWidth: 2,
                  strokeColor: color,
                ),
              )
            );
          }).toList();
        }
```

**Comment by LuisCampos97** [comment]
Hi @jesusmartinoza,
Thanks for your answer.

But I don't think you understand my problem. 
I want to customize the dot color for each of the barLines. That is, I have 2 LineChartBarData (As you can see from the image above, Blue and Green).

And I want to be able to put the blue dot for the first LineChartData and the green dot for the second LineChartBarData.
It is possible?

Thank you!!!

**Comment by LuisCampos97** [comment]
@imaNNeoFighT Can you help me?

**Comment by imaNNeoFighT** [comment]
Do you want to change the color of the text?

**Comment by LuisCampos97** [comment]
No.. I want to change the color of the points.. I want a green point and a blue point, not two blue points like you see in the image.

**Comment by imaNNeoFighT** [comment]
Aha got it.
